be/src/runtime/tablets_channel.cpp

```diff
@@ -107,12 +107,22 @@ Status TabletsChannel::open(const PTabletWriterOpenRequest& request) {
     _next_seqs.resize(_num_remaining_senders, 0);
     _closed_senders.Reset(_num_remaining_senders);
 
-    RETURN_IF_ERROR(_open_all_writers(request));
-
+    if (!config::enable_lazy_open_partition) {
+        RETURN_IF_ERROR(_open_all_writers(request));
```

Comment:
If we are rolling upgrade doris, then may be sink's works as old version and tablets_channel works as latest version, then load breaks.